src/Preferences.cc

```diff
@@ -790,6 +794,8 @@ void Preferences::create(QStringList colorSchemes)
     instance->colorSchemeChooser->clear();
     instance->colorSchemeChooser->addItems(renderColorSchemes);
     instance->init();
+    instance->AxisConfig->init();
+    //instance->ButtonConfig->init();
```

Comment:
This seems to have moved to mainwin.cc, so I'll remove it here.

Reply:
this was mend as a reminder going back to #2261.

It would make sense to initialize all preferences at the same time and place.
For  #2261 (automatically generate the actions for the button comboxes), I had to move the initialization of the ButtonConfig to mainwin.cc, as we need the mainwindow to access its actions (incl. translations).
So, this was a reminder that ButtonConfig has to be initialized, but can't until later.

I could have written a comment, explaining that, but... removing the commented out line is ultimately better as any type of comment (incl. commented out lines of code) out dates as the source code around it changes.